templates/tfengine/recipes/devops.hcl

```diff
@@ -231,7 +238,7 @@ template "root" {
 {{end}}
 
 # At least one trigger is specified.
-{{if and (has . "cicd") (or (has .cicd "validate_trigger") (has .cicd "plan_trigger") (has .cicd "apply_trigger"))}}
+{{if and (has . "cicd") (or (has .cicd.triggers "validate") (has .cicd.triggers "plan") (has .cicd.triggers "apply"))}}
```

Comment:
Nit: Would look cleaner with a `hasOne` function, so like:
```
{{if and (has . "cicd") (hasOne .cicd.triggers "validate" "plan" "apply")}}
```

Reply:
created https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/issues/485 and added TODO